### ISSUE TITLE
Add Ability for More Information/Longer Description

### ISSUE THREAD

**Issue by r3volution11** [insidegui/Sharecuts#5]
As shortcuts grow their descriptions will become more and more important to discern one from another. My assumption is that this will mean descriptions will become longer than 15-20 words.

**Comment by insidegui** [comment]
Yeah I think we should add another field for a longer description which could include setup instructions for more complicated shortcuts
